src/main/telemetry/telemetry.h

```diff
@@ -59,5 +59,4 @@ bool telemetryDetermineEnabledState(portSharing_e portSharing);
 
 void telemetryUseConfig(telemetryConfig_t *telemetryConfig);
 
-#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM)
-
+#define TELEMETRY_SHAREABLE_PORT_FUNCTIONS_MASK (FUNCTION_TELEMETRY_FRSKY | FUNCTION_TELEMETRY_LTM | FUNCTION_TELEMETRY_CRSF)
```

Comment:
@mikeller , I must admit I don't fully understand the sharable aspects of telemetry. What criteria does a telemetry implementation have to satisfy to be shareable?

In `io/serial.c` it states:
```
    /*
     * rules:
     * - 1 MSP port minimum, max MSP ports is defined and must be adhered to.
     * - MSP is allowed to be shared with EITHER any telemetry OR blackbox.
     * - serial RX and FrSky / LTM telemetry can be shared
     * - No other sharing combinations are valid.
     */
 ```
So what is special about FrSky and LTM telemetry that allow them to be shared with serial RX? For example, why cant Mavlink telemetry be shared? Is it because when it was added it was not added to the shared flags, and the comment wasn't updated? Or is there something about Mavlink that prevents it being shared?

As it currently stands CRSF telemetry is half duplex with CRSF RX, that is it shares the Rx pin on the FC. However there is nothing inherent in the CRSF telemetry that requires it to output to Rx, it could be set up in its initialisation to output to the Tx pin and thus, presumably, be sharable. When CRSF telemetry is half-duplex on Rx it does some checking to ensure the telemetry frames are not sent when there is an RX frame coming in, but if the CRSF telemetry was being output to the Tx pin, then that checking would simply not be called (or more probably it would be implemented as a call via a function pointer to a function that does nothing).

The above is not implemented yet, but it is planned. So if all that is working, does CRSF telemetry satisfy the criteria for being shareable?